app/components/UI/SelectOptionSheet/SelectOptionSheet.test.tsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import renderWithProvider from '../../../util/test/renderWithProvider';
+import SelectOptionSheet from './';
+import { iSelectOption } from './types';
+import {
+  NavigationProp,
+  ParamListBase,
+  useNavigation,
+} from '@react-navigation/native';
+import { fireEvent } from '@testing-library/react-native';
+import { SELECT_DROP_DOWN } from './constants';
+
+jest.mock('../../../core/Engine', () => ({
+  context: {
+    colors: {},
+  },
+}));
+
+jest.mock('@react-navigation/native', () => ({
+  ...jest.requireActual('@react-navigation/native'),
+  useNavigation: jest.fn(),
+}));
+
+describe('SelectOptionSheet', () => {
+  it('renders correctly', () => {
```

Comment:
S: Snapshots only allow to detect if it changed since last time, but it's not providing any real insurance that the component is rendered as expected. So avoid the "correctly".
```suggestion
  it('render matches the snapshot', () => {
```